comptests/s3/test_rest_api_rgw.py

```diff
@@ -256,16 +333,645 @@ def test_user_create_36665(self):
+        assert status == HTTPStatus.OK, "Not able to Get user Info. Test Failed"
+        self.log.info("Get user info output: %s", user_info)
+        self.created_users.append(user_params)
+        self.log.info(
+            "Step 3: Creating a new IAM user with same"
+            "user defined access key %s", str(a_key))
+        loop = asyncio.get_event_loop()
+        status, user_info = loop.run_until_complete(self.obj.create_user(user_params2))
+        assert status == HTTPStatus.CONFLICT, "Didnt get the required error status. Test Failed"
+        self.log.info("END : %s", test_case_name)
+
+    @pytest.mark.api_user_ops
+    @pytest.mark.tags('TEST-37299')
+    def test_user_create_37299(self):
+        """Test create user with user defined access key value with special characters."""
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("START: Test create user with user defined "
+                      "access key value with special characters .")
+        user_name = f"{self.user_name_prefix}{str(time.perf_counter_ns()).replace('.', '_')}"
+        a_key = 'ABC#&@$%i?*DEFGH'
+        user_params = {
+            'display-name': user_name,
+            'uid': user_name,
+            'access-key': a_key
+        }
+        self.log.info(
+            "Step 1: Creating a new IAM user with name :%s and "
+            "user defined access key with special character :%s", str(user_name), str(a_key))
+        loop = asyncio.get_event_loop()
+        status, user_info = loop.run_until_complete(self.obj.create_user(user_params))
+        assert status == HTTPStatus.OK, "Not able to create user. Test Failed"
+        self.log.info("Created user details: %s", user_info)
+        self.log.info(
+            "Step 2: Verifying that new IAM user is created successfully")
+        status, user_info = loop.run_until_complete(self.obj.get_user_info(user_params))
+        assert status == HTTPStatus.OK, "Not able to Get user Info. Test Failed"
+        self.log.info("Get user info output: %s", user_info)
+        self.created_users.append(user_params)
+        self.log.info(
+            "Step 3: Using user %s credentials run IOs", user_name)
+        status = self.run_io_using_newuserscredentials(user_info)
+        assert status, "Got Error running IO operations"
+        self.log.info("END : %s", test_case_name)
+
+    @pytest.mark.api_user_ops
+    @pytest.mark.tags('TEST-37300')
+    def test_user_create_37300(self):
+        """Test create user with user defined secret key."""
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        self.log.info("START: Test create user with user defined secret key.")
+        user_name = f"{self.user_name_prefix}{str(time.perf_counter_ns()).replace('.', '_')}"
+        s_key = 'PQRST'
```

Comment:
Keys can be base64 encoded,  large in size /counterfeit  if you are thinking of testing the user inputs with different string types.